### DISCUSSION TITLE
Welcome to Obtainium Discussions!

### DISCUSSION THREAD

**Discussion by ImranR98** [ImranR98/Obtainium#518] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to ask general questions that are neither bugs nor feature requests.

**Comment by ghost** [comment]
Hi, so I asked you in an issue why don't you use GitHub Actions, I hope you'll answer me here.

**Reply by ImranR98** [reply]
Just haven't taken the time to look into it - will do so at some point.

**Reply by ghost** [reply]
Alright, thank you.

What about auto updates ? I think you are getting closer to implement this feature, am I right ? Because I saw today's update has solved many problems.

**Reply by ImranR98** [reply]
No idea how close we are to that. The new plugin does solve some issues related to getting the install result, and it will probably be able to do silent updates as well. The main thing I'm worried about is getting it to work in a background process instead of the main foreground Flutter process - that doesn't work at all and (based on some searching) it may be a Flutter limitation that there's no way around.

**Reply by ghost** [reply]
I see, but whatever the result will be, I speak for myself when I say I will be more than satisfied if you did implement it to work in the background or the foreground.

You already did your best to provide such a great app that makes us able to do things we could never do before in a convenient way.

Thank you a lot.

**Comment by Damadduke** [comment]
Hello I just started using Obtainium and really like it. How can I make donation to you?

**Comment by zen0bit** [comment]
❤️ Obtainium

**Comment by E0697** [comment]
Is it okay to request sources in issues?

**Comment by katastrophe92** [comment]
Hello Obtainium community! Thanks @ImranR98 for this noice repository☺! 

Can I import **_only_** my '' Android APK's '' GitHub list into Obtainium? That would be awesome. 
If I login into GitHub inside Obtainium App gives any benefit?

Hope someone can answer my questions. Thanks!

**Reply by Damadduke** [reply]
You can export your app list from Obtainium to import into another installation of it. I recommend doing this as it makes any recovery or re installation much, much easier.

**Reply by katastrophe92** [reply]
I know this. Only a GitHub Starred list containing only android apps I wanted to import to Obtainium. Is this possible @ImranR98

**Reply by ImranR98** [reply]
@katastrophe92 not possible currently - you can open an issue for it

**Comment by nodeunit** [comment]
Hello, I was looking through the readme and saw that Steam (as in valve steam) was supported as a source, and I'm confused as to what the use cases are. Does Valve really allow developers to distribute APKs, and if so are there actually any games that do so???

**Reply by ImranR98** [reply]
No, it's for the Android apps from Valve themselves (I think Steam chat and a few others)

**Comment by JDKamalakar** [comment]
Cannot add obtimum to the list of apps
